### ISSUE TITLE
Icon for Plugins

### ISSUE THREAD

**Issue by NtQuery** [x64dbg/x64dbg#291]
As a plugin developer, I want to have a custom icon in the context menu. Just like the Scylla icon.

**Comment by mrexodia** [comment]
damn, you also starting with that? :joy: I'll take a look at it. Probably I can do something with bitmaps.

**Comment by mrfearless** [comment]
lol :D 

Id mentioned it as well last night. could do it a couple of ways, name of plugin.bmp to load from plugins folder, or as a bitmap resource handle passed back to x64dbg from a plugmenuicon call perhaps, or a set list of icons (bitmaps) to choose from only with a menubitmap integer param on menuadd or something like that

**Comment by NtQuery** [comment]
PNG is ok... does not need to be bitmap or ico

**Comment by NtQuery** [comment]
ah let me rephrase it: Please support only PNG. Do not waste your time on bitmap or ICO. PNG is the superior image format. It is really easy to convert BMP or ICO to PNG. Please do not break plugin compatibility. Don't change structs... just add a new method so the plugin can provide the binary buffer for the icon.

**Comment by mrexodia** [comment]
yea, that was exactly my idea :)

2015-04-26 20:53 GMT+02:00 NtQuery <notifications@github.com>:

**Comment by mrexodia** [comment]
support for pretty much any image format you want
